src/controller/CHIPDevice.h

```diff
@@ -297,6 +297,14 @@ class DLL_EXPORT Device : public Messaging::ExchangeDelegate, public SessionEsta
      */
     CHIP_ERROR OpenPairingWindow(uint32_t timeout, PairingWindowOption option, SetupPayload & setupPayload);
 
+    /**
+     *  @brief
+     *    In case there exists an open session to the device, mark it as expired.
+     *
+     * @return CHIP_NO_ERROR on success, an error code otherwise.
```

Comment:
drop the return description (and maybe also the `@brief` empty line): it is obvious that returning CHIP_ERROR will return 'NO ERROR on success, an error otherwise'.